Library/Formula/libghthash.rb

```diff
@@ -0,0 +1,30 @@
+class Libghthash < Formula
+  homepage "http://www.bth.se/people/ska/sim_home/libghthash.html"
+  url "http://www.bth.se/people/ska/sim_home/filer/libghthash-0.6.2.tar.gz"
+  sha1 "436146998c30a31b83adf4f58d978fccd78f2ba5"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+    Dir["#{lib}/*[^.a]"].each do |f|
+      File.rename(f, "#{lib}/" + File.basename(f) + ".dylib")
```

Comment:
Can you please submit a patch doing to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!